### PR TITLE
Update __init__.py

### PR DESCRIPTION
**Pull request by rishubhjain**
tendrl-bugid: Tendrl/monitoring-integration#214

### CODE REVIEW

**Comment by rishubhjain** [comment]
@GowthamShanmugam Please check this code with the changes done on the testing machine

**Comment by GowthamShanmugam** [comment]
@r0h4n i am testing this

**Comment by GowthamShanmugam** [comment]
@r0h4n verified

File: tendrl/monitoring_integration/flows/new_cluster_dashboard/__init__.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ def run(self):
         self.upload_alert_dashboard(cluster_id)
 
     def upload_alert_dashboard(self, integration_id=None):
+        self.global_flag = False
```
Comment (r0h4n) [inline tendrl/monitoring_integration/flows/new_cluster_dashboard/__init__.py]: global_flag for what? Please use meaningful var names

**Comment by rishubhjain** [comment]
@r0h4n Made the changes, Please review

File: tendrl/monitoring_integration/flows/new_cluster_dashboard/__init__.py [hunk 2]
```diff
@@ -72,6 +73,10 @@ def create_all_dashboard(self, dashboard_name, cluster_detail_list):
                            "with error {0}".format(str(error))})
 
     def create_resource(self, integration_id, cluster_detail_list, resource_type):
+        if reource_type == "volumes":
```
Comment (r0h4n) [inline tendrl/monitoring_integration/flows/new_cluster_dashboard/__init__.py]: There's typo "reource_type" , is this even verified?

Comment (GowthamShanmugam) [reply tendrl/monitoring_integration/flows/new_cluster_dashboard/__init__.py]: @rishubhjain please make sure this function is calling

**Pull request merged**